types/types.go

```diff
@@ -445,8 +445,33 @@ type TraefikLog struct {
 
 // AccessLog holds the configuration settings for the access logger (middlewares/accesslog).
 type AccessLog struct {
-	FilePath string `json:"file,omitempty" description:"Access log file path. Stdout is used when omitted or empty" export:"true"`
-	Format   string `json:"format,omitempty" description:"Access log format: json | common" export:"true"`
+	FilePath         string           `json:"file,omitempty" description:"Access log file path. Stdout is used when omitted or empty" export:"true"`
+	Format           string           `json:"format,omitempty" description:"Access log format: json | common" export:"true"`
+	HeaderRedactions HeaderRedactions `json:"headerredactions,omitempty" description:"List of names of sensitive headers to redact" export:"true"`
```

Comment:
@ldez @emilevauge does `export` specify if this should be exported for usage statistics? If so, is it okay to include redacted headers?

Personally I'm not sure.